nixos/modules/hardware/video/nvidia.nix

```diff
@@ -394,6 +394,9 @@ in {
             modules = [nvidia_x11.bin];
             display = !offloadCfg.enable;
             deviceSection =
+              ''
+                Option "SidebandSocketPath" "/run/nvidia-xdriver/"
```

Comment:
Did you verify that this works as intended? The name of the option suggests it expects a full socket path, not a directory.